conf/ledgersmb-nginx.template

```diff
@@ -0,0 +1,63 @@
+##
+# LedgerSMB server configuration
+#
+server {
+        listen 80 default_server;
+        listen [::]:80 default_server;
+
+        #Set to LSMB installed directory
+        root [ROOT];
+
+        index login.pl;
+
+        #Adjust to your liking
+        #server_name ubuntu-vm;
+
+	#Try the asked file directly, pass to Plack if not found
+        location / {
+            try_files $uri @plack;
+        }
+
+        location @plack {
+            proxy_set_header Host $http_host;
+            proxy_set_header X-Forwarded-Host $http_host;
+            proxy_set_header X-Real-IP $remote_addr;
+            proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
```

Comment:
They are not needed in Apache. They are required by NGinx because it
lacks the proxypassreverse statement

Le 2016-04-22 14:47, Erik Huelsmann a écrit :
>
> In conf/ledgersmb-nginx.template
> <https://github.com/ehuelsmann/LedgerSMB/pull/240#discussion_r60785489>:
>
> > +
> > +        index login.pl;
> > +
> > +        #Adjust to your liking
> > +        #server_name ubuntu-vm;
> > +
> > +	#Try the asked file directly, pass to Plack if not found
> > +        location / {
> > +            try_files $uri @plack;
> > +        }
> > +
> > +        location @plack {
> > +            proxy_set_header Host $http_host;
> > +            proxy_set_header X-Forwarded-Host $http_host;
> > +            proxy_set_header X-Real-IP $remote_addr;
> > +            proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
>
> Nice headers. Is it possible to add these to Apache and lighthttpd too
> (or are they automatic there?)
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/ehuelsmann/LedgerSMB/pull/240/files/a6636f6dc8d0aad08544168b52862b70c41ba586#r60785489>
>